src/core/lib/iomgr/tcp_windows.cc

```diff
@@ -74,12 +74,26 @@ static grpc_error* set_dualstack(SOCKET sock) {
              : GRPC_WSA_ERROR(WSAGetLastError(), "setsockopt(IPV6_V6ONLY)");
 }
 
+static grpc_error* enable_loopback_fast_path(SOCKET sock) {
+  int status;
+  uint32_t param = 1;
+  DWORD ret = 0;
+  status = WSAIoctl(sock, /*SIO_LOOPBACK_FAST_PATH==*/_WSAIOW(IOC_VENDOR, 16),
+                    &param, sizeof(param), NULL, 0, &ret, 0, 0);
+  return status != SOCKET_ERROR || status == WSAEOPNOTSUPP
```

Comment:
I believe that you need to call `WSAGetLastError` to see the more detailed `WSAEOPNOTSUPP`. `WSAIoctl` will always just fail with `SOCKER_ERROR`.